sources/packages/backend/apps/queue-consumers/src/controllers/health-check/health.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Controller, Get } from "@nestjs/common";
+import { RedisOptions, Transport } from "@nestjs/microservices";
+import {
+  HealthCheck,
+  MicroserviceHealthIndicator,
+  HealthCheckService,
+  TypeOrmHealthIndicator,
+} from "@nestjs/terminus";
+import { ConfigService } from "@sims/utilities/config";
+import { DataSource } from "typeorm";
+
+@Controller("health")
+export class HealthController {
+  constructor(
+    private readonly configService: ConfigService,
+    private healthCheckService: HealthCheckService,
+    private typeOrmHealthIndicator: TypeOrmHealthIndicator,
+    private microservice: MicroserviceHealthIndicator,
+    private dataSource: DataSource,
+  ) {}
+
+  /**
+   * Check the health of the Queues.
+   * @returns the status of the health for Queues, with info or error and details.
+   */
+  @Get()
+  @HealthCheck()
+  async check() {
+    return this.healthCheckService.check([
+      () =>
+        this.typeOrmHealthIndicator.pingCheck("Postgres", {
```

Comment:
Thanks for adding this. Should we make the description like `sims-db` or `sims-database` that would represent exactly what the `this.dataSource` is representing?